app/controllers/api_controller/providers.rb

```diff
@@ -5,9 +5,14 @@ module Providers
     CREDENTIALS_ATTR  = "credentials"
     AUTH_TYPE_ATTR    = "auth_type"
     DEFAULT_AUTH_TYPE = "default"
-    ENDPOINT_ATTRS    = %w(hostname ipaddress port)
+    ENDPOINT_ATTRS    = %w(hostname ipaddress port connection_configurations_array).freeze
```

Comment:
> I'm not crazy about the name connection_configurations_array, if we need it, maybe something like connection_configuration_definitions or simply connection_definitions. @imtayadeway thoughts ?

@imtayadeway @abellotti chenged to `connection_definitions`, I like this name :-) but I'm open to change ...